distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,65 @@
+:title: Federated Identity
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated sources during queries
+
+Identity federation is the sharing of a user's identity and attributes among separate systems. In
+{branding}, it is used when performing federated queries - the user's identity is attached to outgoing
+requests, allowing federated sources to run queries as that user. This is critical because DDF must
+make authorization decisions for all queries, not just those initiated by the local node.
+
+To represent a user's identity when federating, {branding} uses a SAML assertion. This is the format
+used internally to represent all subjects (even unauthenticated subjects). When federating over REST,
+the assertion is simply attached as an HTTP header to the query request, and extracted by the
+federated source when the request is received. If the assertion is signed by a trusted cert, the
+federated source will then run the query as the user.
+
+The figure below shows a federated query between two instances of {branding} that support
+federated identity.
+
+[ditaa, federated_identity, png, ${image-width}]
+....
+                  /---------------------------\           /--------------------------\
+                  | cDEF       DDF            |           |     Federated Source     |
```

Comment:
❓ I'm not sure if it's possible to put a branding inside of a diagram or not, if it is I'd recommend to put one here. 
I'm fairly certain you can't though.

Reply:
I tried, but couldn't get it to work. If somebody knows, I'd love to hear it

Reply:
Yeah I'm thinking it's not possible.

Reply:
I love the diagram.

Reply:
Line 16. Certs are used to do the signing but are not agents who initiate or make signing decisions. "Trusted entity" is probably the right phrase:

"If the assertion is signed by a trusted entity, the Federated Source will then run the query as the user."

Also, eliminate "will then" to simply the sentence. 
Change "runs" to "processes" or "accepts" 

`If the Assertion is signed by a trusted entity,  the Federated Source processes the request.`